hieradata/common.yaml

```diff
@@ -79,6 +79,13 @@ backup::offsite::jobs:
     hour: 8,
     minute: 13,
     # No encryption because of size and sensitivity
+  'govuk-cdn-logs':
```

Comment:
No, it's a good point. I'm not sure that the match for `"/data/backups/logs-cdn-1.management.%{hiera('app_domain')}/*"` is the best way of doing it. It could also be "/data/backups/*%{hiera('govuk::apps::govuk_cdn_logs_monitor::processed_data_dir')}". Not sure which is nicer.